src/cljs/vanilla/add_widget.cljs

```diff
@@ -325,6 +329,10 @@
                                   :data-format/y]
                       :icon "/images/bubble-widget.png"})
 
+  (def services (rf/subscribe [:services]))
+  @services
+  (vec (remove #(= "carousel-service" (:keyword %)) @services))
```

Comment:
lets do this by removing this service from the vanilla_db so we don't have this special case

Reply:
Ahhh, yeah good idea. I (wrongly) assumed doing that would break subscriptions or something else, but since all of the carousel is dummy right now, it was no problem.  Done!